### DISCUSSION TITLE
Notify users using when they are viewing the development docs

### DISCUSSION THREAD

**Discussion by jorgepiloto** [pyansys/actions#119] [Polls]
Hi all 👋🏽,

I am raising a poll about whether or not we should add an announcement when users are reading the development documentation. This feature was suggested last week and we would love to know your thoughts about whether to implement it or not.

The announcement would be similar to the one showing under the top navigation bar when users are viewing an outdated version of the documentation:

![Screenshot 2022-12-13 at 17-22-41 PyMAPDL Documentation 0 62 3 — PyMAPDL](https://user-images.githubusercontent.com/28702884/207387884-b980500c-2ee8-40c7-8ea6-9ffbc5bdc78d.png)

**Comment by da1910** [comment]
I'd suggest making it a different color, but it's certainly useful to know!

**Reply by germa89** [reply]
Kind of agree on that... Yellow?? Orange?? Green??

**Reply by da1910** [reply]
Looks like numpy and scipy use yellow for dev, but they color the version selector dropdown rather than adding a banner

**Reply by PProfizi** [reply]
I agree with that, maybe yellow like an "under construction" sign

**Reply by da1910** [reply]
![image](https://user-images.githubusercontent.com/12086549/207391510-5524e566-03e3-4cd1-be0f-601d66a3828e.png)
![image](https://user-images.githubusercontent.com/12086549/207391546-f3dee3d8-c0a7-425d-b2cf-10387b02b430.png)
![image](https://user-images.githubusercontent.com/12086549/207391565-0a8b861b-99aa-40b2-89a3-8ba93f86e9fb.png)

**Reply by PProfizi** [reply]
Not sure you'd have to redirect to the latest stable though if that's something complicated to do.

**Reply by greschd** [reply]
@da1910 not sure if that's the suggestion, but I think at least the "outdated" warning should be more prominent than _just_ color-coding. I think the banner works well for that. For the dev version, maybe color is enough. 

If we go for a banner, a "friendly" color like green would also be sufficient IMO. It would stand out simply by the fact of being a banner (as opposed to no banner on stable).

**Reply by germa89** [reply]
I agree with @greschd we need the banner for the outdated versions.

Regarding the color for `dev` I would propose yellow/orange. Green might look like "everything is fine", which might not be the case.

**Reply by da1910** [reply]
I agree about the banner, the button coloring isn't clear enough for me, I was only including the screenshots to indicate the color choices!

**Reply by timdansys** [reply]
Regarding semantic color usage, we use the following approach:

- **Red:** Errors, dangerous or destructive actions or statements
- **Yellow:** Warnings, caution
- **Blue:** Informative statements
- **Green:** Success

With that in mind, the proposed message that the user is reading the development documentation would be a warning or a caution (yellow) because the user may not know this isn't the latest stable version.

We also do not want to present a warning message with a dead end - we need to provide a link to the latest stable version, unless it's obvious how to get to the latest stable version.
